code/modules/mob/living/living_defines.dm

```diff
@@ -58,3 +58,6 @@
 	var/unique_name = 0 //if a mob's name should be appended with an id when created e.g. Mob (666)
 
 	var/list/butcher_results = null
+
+	var/stun_absorption = FALSE //If all incoming stuns are being absorbed

```

Comment:
The definition of hard & shitcode.

We're better than oldcoders, you can atleast do this semi decently.